hack/ci/e2e.sh

```diff
@@ -87,15 +85,14 @@ kubectl -n source-system delete -f "${ROOT_DIR}/config/testdata/helmchart-values
 
 echo "Setup Minio"
 kubectl create ns minio
-helm repo add minio https://helm.min.io/ --force-update
-helm upgrade minio minio/minio --wait -i \
+helm upgrade minio oci://registry-1.docker.io/bitnamicharts/minio --wait -i \
     --version "${MINIO_HELM_VER}" \
+    --timeout 10m0s \
```

Comment:
Adjusted timeout due to error in [pipeline](https://github.com/fluxcd/source-controller/actions/runs/7046816530/job/19179363913). Pipeline passing now with new timeout

> Release "minio" does not exist. Installing it now.
Pulled: registry-1.docker.io/bitnamicharts/minio:12.10.3
Digest: sha256:530b2db8981088ab479a4503fa88e211f6808038f6b0177e4715d3d44f7510b2
Error: timed out waiting for the condition